### PR TITLE
Change binary name to bizfly-backup

### PR DESCRIPTION
**Pull request by greatbn**

### CODE REVIEW

File: scripts/linux.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ check_distribution(){
         sudo apt-get install -y jq curl
         echo "support"
     # For CentOS/RHEL 6,7,8
-    elif [[ $distribution_raw == *rhel* || -f "/etc/redhat-release" ]] ; then                       
+    elif [[ $distribution_raw == *rhel* || -f "/etc/redhat-release" ]] ; then
```
Comment (cuonglm) [inline scripts/linux.sh:13]: Don't combine this to complex expression. Use:

```
[[ $distribution_raw == *rhel* ]] || [[ -f "/etc/redhat-release" ]] 
```

File: scripts/linux.sh [hunk 2]
```diff
@@ -36,7 +36,7 @@ get_lastest_download_url(){
     else
         i=0
         while [ $i -lt $length ]
```
Comment (cuonglm) [inline scripts/linux.sh:38]: This needs double quotes: `while [ "$i" -lt "$length" ]`

File: scripts/linux.sh [hunk 3]
```diff
@@ -57,51 +57,52 @@ download_agent(){
         else
             curl -Ls $(get_lastest_download_url) --output "bizfly-backup.tar.gz"
```
Comment (cuonglm) [inline scripts/linux.sh:58]: $(get_lastest_download_url)

**Review by cuonglm — CHANGES_REQUESTED** [review]

**Review by QuocCuong97 — COMMENTED** [review]
OK anh ạ

**Pull request merged**